load-test-framework/src/main/java/com/google/pubsub/flic/common/MessagePacketProto.java

```diff
@@ -0,0 +1,745 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Do we need to include the generated file? I would think the Maven compilation would take care of generating it on the fly.